ingest/api/std_name_units.json

```
@@ -0,0 +1,481 @@
+{
+    "diffuse_downwelling_shortwave_flux_in_air": {
+        "unit": "W/m2",
+        "alias": [
+            "W m-2",
+            "Wm-2"
+        ],
+        "symbol": "W/m2"
+    },
+    "downwelling_longwave_flux_in_air": {
+        "unit": "kJ/m2",
+        "alias": [
+            "kJm-2",
+            "kJ m-2"
+        ],
+        "symbol": "kJ/m2"
+    },
+    "duration_of_sunshine": {
+        "unit": "s",
+        "symbol": "s",
+        "conversion": {
+            "minutes": {
+                "mul": 60
+            }
+        }
+    },
+    "equivalent_reflectivity_factor": {
+        "unit": "dB",
+        "symbol": "dB"
+    },
+    "integral_wrt_time_of_surface_downwelling_longwave_flux_in_air": {
+        "unit": "Ws/m2",
+        "alias": [
+            "W s m-2",
+            "Ws m-2",
+            "Wsm-2"
+        ],
+        "symbol": "Ws/m2"
+    },
+    "latitude": {
+        "unit": "degrees",
+        "symbol": "degrees",
+        "alias": [
+            "deg",
+            "degree"
+        ]
+    },
+    "longitude": {
+        "unit": "degrees",
+        "symbol": "degrees",
+        "alias": [
+            "deg",
+            "degree"
+        ]
+    },
+    "low_type_cloud_area_fraction": {
+        "unit": "octas",
+        "symbol": "octas"
+    },
```

```suggestion
    "low_type_cloud_area_fraction": {
        "unit": "oktas",
        "symbol": "oktas",
        "alias": [
            "octas"
        ]
    },
```

https://en.wikipedia.org/wiki/Okta